edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -133,8 +133,21 @@ def ComputeResult(self, value, cond, value2):
         """
         ivalue = value
         ivalue2 = value2
+        if isinstance(value, str):
+            ivalue = value.strip("\"")
+        if isinstance(value2, str):
+            ivalue2 = value2.strip("\"")
         # convert it to interpretted value
 
+        if (cond.upper() == "OR"):
+            return value or value2
```

Comment:
Are these supposed to be value or ivalue?

Reply:
Did not mean to approve.

Reply:
Hmmm. Maybe. ivalue is meant to be the integer representation of the value. So maybe. We need to convert to a number.

Reply:
But then we get to weird situations. What does !if 1 and TRUE mean?

Reply:
Alright- I decided that we should use ivalue in this situation